### DISCUSSION TITLE
Android: Changes to textbox from codebehind are not reflected to viewmodel

### DISCUSSION THREAD

**Discussion by DrHardReset** [unoplatform/uno#16937] [General]
In my application [RoMi](https://github.com/DrHardReset/RoMi) I am setting the Text property of a TextBox in the [code behind class](https://github.com/DrHardReset/RoMi/blob/master/RoMi/Presentation/MainPage.xaml.cs).
In the windows build this change is correctly reflected to the corresponding viewmodel.
The android build behaves different and does not reflect the change to the viewmodel.

Is this a bug or am I missing something?

View:
```
<TextBox x:Name="PdfUrl"
	 PlaceholderText="Enter MIDI PDF URL"
	 Text="{Binding DownloadPdfUrl, Mode=TwoWay}"/>
```

View codebehind:
```
private void CobDeviceSelection_SelectionChanged(object sender, SelectionChangedEventArgs e)
{
    if (e.AddedItems.Count == 0)
    {
        return;
    }

    PdfUrl.Text = ((KeyValuePair<string, string>)e.AddedItems[0]).Value;
}
```

Viewmodel:
```
[ObservableProperty]
private string downloadPdfUrl = string.Empty;
```

**Comment by agneszitte** [comment]
cc @jeromelaban, @Youssef1313

**Comment by Youssef1313** [comment]
I converted this to an issue #16951.
